docs/data/base/components/slider/slider.md

```diff
@@ -152,6 +152,14 @@ You may not need to use hooks unless you find that you're limited by the customi
 
 ## Customization
 
+### Vertical
+
+Slider components can be arranged vertically as well as horizontally.
+
+When vertical, you must set `orientation="vertical"` on the `<Slider />` component so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal slider).
```

Comment:
Just one tiny grammar fix, otherwise this looks good to me!

```suggestion
When vertical, you must set `orientation="vertical"` on the `<Slider />` component so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal sliders).
```

Reply:
@samuelsycamore resolved in this commit https://github.com/mui/material-ui/pull/37357/commits/f3b9cb6a6d5c987af84914cd7c629d0592f65ee2